HW8/task4.py

```diff
@@ -0,0 +1,10 @@
+def filter_int(elem):
+    # check type of elements
+    if type(elem) == int or type(elem) == float:
+        return elem
```

Comment:
not correct here it returns an element of the sequence and `filter` check if it is bool, so 0 element will be omitted. So boolean value should be returned
```suggestion
    return isinstance(elem, (int, float))
```

Reply:
i am changed this mistake